src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/LocationBasedReservedIPTests.cs

```diff
@@ -68,16 +68,14 @@ public void CreateReservedIPThenPaaSVM()
         {
             try
             {
-                string reservedIpName1 = "ResrvdIP1";
-                string reservedIpName2 = "ResrvdIP2";
+                string reservedIpName1 = Utilities.GetUniqueShortName("ResrvdIP1"); ;
```

Comment:
In the future, consider using the existing utilities in the test frameowrk for generating names - this will store names in recorded mocks